packages/ui/src/components/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -98,13 +93,47 @@ export const ConsumerGroupResetOffset: FunctionComponent<
     new_offset: t("consumerGroup.new_offset"),
   };
 
+  const onSelectAllConsumer = (isSelecting = true) => {
+    setSelectableConsumer(
+      selectableConsumer.map((consumer) => {
+        consumer.selected = isSelecting;
```

Comment:
Never mutate a data structure, especially one that represents data coming from an external source! Please keep track of the selected items in a different state.

https://javascript.plainenglish.io/javascript-mutable-vs-immutable-1efb662d78c8

Reply:
Done